### PR TITLE
feat(ci): upgrades ci to run tests using python 3

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

This PR upgrades ftw to 1.2.3, which has py3 support.

### CODE REVIEW

**Review by fgsch — APPROVED** [review]
👍

**Comment by fgsch** [comment]
Thank you!

This is p ending the ftw release (should happen in the next few hours).

File: tests/regression/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-ftw==1.1.7
+ftw==1.2.2
```
Comment (fgsch) [inline tests/regression/requirements.txt:1]: This will work but we might want to use 1.2.3

Comment (fzipi) [reply tests/regression/requirements.txt]: Updated based on pypi release.

**Comment by dune73** [comment]
@fzipi : This has some tests failing. Otherwise it would be good to go, would not it?

And are the failing tests failing because of the changes to FTW?

**Comment by fzipi** [comment]
@dune73 Yes, this is becauseof the changes regarding the py3 full support. Some things are handled differently by py2 and py3, in particular string vs. bytes and encoding.

With @fgsch were in the mix for getting a proper version but it might be a breaking change for some users (_might be_). So we cannot merge this one yet.

**Comment by dune73** [comment]
OK. So this is pending for the time being. Thank you.

**Comment by dune73** [comment]
Any update on this front @fzipi and @fgsch?

**Comment by fzipi** [comment]
Looks like python 3.9 fails with the 1.2.3 version, but 3.8 works. Will then fix the pipeline run to 3.8 for now.

**Comment by dune73** [comment]
Great work @fzipi. Thank you very much.

**Review by fgsch — APPROVED** [review]
Yay!

**Comment by fzipi** [comment]
Finally, it is working. We might want to merge! @dune73

**Pull request merged**